test/jasmine/tests/mapbox_test.js

```diff
@@ -475,6 +492,22 @@ describe('mapbox plots', function() {
         });
     });
 
+    it('should be able to update the access token', function(done) {
+        var promise = Plotly.relayout(gd, 'mapbox.accesstoken', 'wont-work');
+
+        promise.catch(function(err) {
+            expect(gd._fullLayout.mapbox.accesstoken).toEqual('wont-work');
+            expect(err).toEqual(new Error(constants.mapOnErrorMsg));
```

Comment:
@lpan So, what are you proposing?

> one thing we can do is to make our token validation impeccable 

I'd vote :-1: for adding token validation in plotly.js. Making one request to validate the token and then one to plot the map sounds like an overkill. No?



> For now in my opinion when the user supplies a bad token/style we catch the error then pop up an error dialogue telling them what is going on.

... as currently off this branch, correct?